superset-frontend/src/visualizations/TimeTable/TimeTable.jsx

```diff
@@ -129,7 +129,16 @@ const TimeTable = ({
         sortType: (rowA, rowB, columnId) => {
           const rowAVal = rowA.values[columnId].props['data-value'];
           const rowBVal = rowB.values[columnId].props['data-value'];
-          return rowAVal - rowBVal;
+          if (typeof rowAVal === 'number' && typeof rowBVal === 'number') {
```

Comment:
This could be simplified as:

```ts
return (
  (rowAVal == null) - (rowBVal == null) ||
  Number.isNaN(rowAVal) - Number.isNaN(rowBVal == null) ||
  Number(a) - Number(b)
);
```

Reply:
@ktmud thank you for the simplified expression, but i feel it is a little hard to digest. I would prefer straightforward expression, given there is no significant space or complexity optimization. And after user research, we might want to add extra logic to always push null values at the bottom (instead of sorting as smallest), i think my original expression is easier to update and understood.

Reply:
My proposed expression already included logics to sort `null` and `NaN` to the bottom.

Reply:
Push null and NaN to the bottom is not decided yet. I think we will run user research to decide sort or push.